LICENSES/BSD-2-Clause.txt

```diff
@@ -0,0 +1,9 @@
+Copyright (c) <year> <owner> 
```

Comment:
These left blank on purpose (ditto the other two BSD licenses)

Reply:
Yeah this is expected
https://reuse.software/faq/#license-templates
> Instead of inserting your copyright notice into the license text itself, you add copyright statements to your project’s files following the normal REUSE best practices.